util/tracing/Makefile

```diff
@@ -30,8 +31,10 @@ install: trace_to_csv trace_to_chrome trace_to_influxdb
 	mv trace_to_csv $(INSTALL_PREFIX)
 	mv trace_to_chrome $(INSTALL_PREFIX)
 	mv trace_to_influxdb $(INSTALL_PREFIX)
-	ln -f -s launch-fedsd.sh $(INSTALL_PREFIX)/fedsd
-	chmod +x launch-fedsd.sh
+	cp ./visualization/fedsd.py $(INSTALL_PREFIX)
+	cp ./visualization/fedsd_helper.py $(INSTALL_PREFIX)
+	ln -f -s $(INSTALL_PREFIX)/fedsd.py $(INSTALL_PREFIX)/fedsd
```

Comment:
Why use the `-f` flag?

Reply:
It is useful when one installs the tracing utilities with an `INSTALL_PREFIX` (that does not need `sudo`, and that is the path), and you are updating the utilities.
Doesn't this make sense?